### PR TITLE
Translate suggestions.md

### PR DESCRIPTION
**Pull request by Fridolin012**
## Type of PR

- [X] Translation update / creation
- [ ] Original file creation
- [ ] Original file content (not typo) update
- [X] Original file typo update

## Checks

If you have changed the content (/ meaning) of an original file, please confirm:

- [ ] Yes, I have copied the updated section to all translations of this file manually.

## Notes

Please enter additional information here.

### CODE REVIEW

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/suggestions.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+<ModuleOverview moduleName="suggestions" />
+
+## Funktionen {#features}
+
+* Lasse deine Nutzer Vorschläge auf Discord einreichen, indem sie
+    * Befehle nutzen
+    * Nachrichten in einen ausgewählten Kanal schicken (optional)
+* Nimm Vorschläge von Nutzern an / lehne sie ab
+* Lasse Nutzer für Vorschläge abstimmen, indem sie Reaktionen nutzen
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/suggestions.md:13]:
```suggestion
* Lasse Nutzer über mit Reaktionen Vorschläge abstimmen
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/suggestions.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+* Öffne automatisch Threads unter neuen Vorschlägen (optional)
+
+## Einrichtung {#setup}
+
+1. Erstelle für die Vorschläge einen Kanal (oder wähle einen aus). Der Bot benötigt folgende Rechte auf diesen: "Kanal
+   anzeigen" ("View channel"), "Nachrichten senden ("Send messages") und "Nachrichtenverlauf sehen" ("View message history")
+2. Stelle sicher, dass [Berechtigungen](./../../slash-commands) auf `/manage-suggestions` richtig konfiguriert sind, um zu verhindern,
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/suggestions.md:20]:
```suggestion
2. Stelle sicher, dass [Berechtigungen](./../../slash-commands) für `/manage-suggestions` richtig konfiguriert sind, um zu verhindern, 
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/suggestions.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+4. [Konfiguriere](#configuration) das Modul.
+5. Lade die Konfiguration neu, um Änderungen zu übernehmen.
+
+## Nutzung {#usage}
+
+* Nutzer können einen neuen Vorschlag mit `/suggestion` einreichen. Neue Vorschläge werden in den konfigurierten Kanal gesendet.
+* Wenn aktiviert, werden Nachrichten, die in den Vorschläge-Kanal gesendet werden, automatisch zu Vorschlägen konvertiert.
+* Andere Nutzer können mit den konfigurierten Reaktionen abstimmen und mit Threads zu den Vorschlägen kommentieren.
+* Admins können Vorschläge mit `/manage-suggestions` annnehmen oder ablehnen und optional auch einen Kommentar zu ihrer Entscheidung angeben. Das 
+  wird den Vorschlags-Ersteller, wenn konfiguriert, über PNs informieren.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/suggestions.md:32]:
```suggestion
* Admins können Vorschläge mit `/manage-suggestions` annnehmen oder ablehnen und optional auch einen Kommentar zu ihrer Entscheidung angeben. Drüber 
  wird der Vorschlag-Ersteller, wenn konfiguriert, über PNs informiert.
```

**Review by SCDerox — CHANGES_REQUESTED** [review]

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/suggestions.md (hunk 1 above)
Comment (Fridolin012) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/suggestions.md:13]: That makes no sense

Comment (SCDerox) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/suggestions.md:13]: i'm sorry my brain is faster than I can type (definitely that and not my lack of sleep)

**Review by SCDerox — APPROVED** [review]

**Comment by SCDerox** [comment]
Thanks for your contribution - you have received a payout as part of our [Open Source Developer Program](https://faq.scnx.app/open-source-developer-pool/). You can find details in your SCNX organisation payout history. To pay out the received amount, please follow instructions in your dashboard. If you have any questions, please contact ScootKit staff on [scnx.app/help](https://scnx.app/help).
Cheers ✌️

**Pull request merged**